apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -207,41 +239,70 @@ var TopInstructions = React.createClass({
                 headerHasFocus={false}
                 onClick={this.handleDocumentationClick}
               />}
+            {experiments.isEnabled('resourcesTab') &&
+              <div style={styles.helpTab}>
+                <a
+                  className="instructionsTab"
```

Comment:
When we have classNames that are used only so that we have an easy selector for UI tests, I like to make that clear by prepending the name with `uittest`, i.e. `uittest-instructionsTab`. This way, when someone else is reading the code they can easily understand the purpose of the classname, and don't need to worry (as much) that there's some code elsewhere that is modifying this DOM based on the classname.